addons/l10n_it_edi/models/account_edi_format.py

```diff
@@ -399,24 +399,77 @@ def _import_fattura_pa(self, tree, invoice):
         invoices = self.env['account.move']
         first_run = True
 
+        buyer_seller_info = {
+            True: {
```

Comment:
It could be, then when we access the dictionary, we could do sth like:
```py
company_key, partner_key = ('buyer', 'seller') if incoming else ('seller', 'buyer')
```

Reply:
I changed the keys to 'buyer' and 'seller' in a new version and used the dedicated variables for company and partner info everywhere.